readme.md

```diff
@@ -15,9 +15,11 @@
 [npm][]:
 
 ```sh
-npm install rehype-autolink-headings
+npm install rehype-autolink-headings rehype-slug
```

Comment:
```suggestion
npm install rehype-autolink-headings
```

`rehype-slug` is not needed, any method for adding ids to headings could be used.